native-interaction/rubix-native.proto

```diff
@@ -72,6 +72,11 @@ message Status {
   bool status = 1;
 }
 
+message GenerateReq {
+  double tokenCount = 1;
+  string did = 2;
```

Comment:
as the discussed, this is just for testing and this api will be removed.

Otherwise, did was supposed to be taken by decoding the access token.